python/examples/cyl-ellipsoid.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import division
+
+import meep as mp
+from meep.geom import Cylinder, epsilon, Ellipsoid, Vector3, Medium
+from meep.simulation import Mirror, Pml, Simulation, no_size
+from meep.source import GaussianSource, Source
+
+
+def main():
+
+    c = Cylinder(radius=3, material=Medium(epsilon_diag=epsilon(3.5)))
+    e = Ellipsoid(size=Vector3(1, 2, 1e20))
+
+    src_cmpt = mp.Hz
+    sources = Source(src=GaussianSource(1, 0.1), component=src_cmpt, center=Vector3())
+
+    if src_cmpt == mp.Ez:
+        symmetries = [Mirror(mp.X), Mirror(mp.Y)]
+
+    if src_cmpt == mp.Hz:
+        symmetries = [Mirror(mp.X, -1), Mirror(mp.Y, -1)]
+
+    sim = Simulation(cell_size=Vector3(10, 10, no_size),
```

Comment:
Can we just allow `0` here rather than `no_size`?  I've come to dislike `no-size` in the Scheme interface, which was an early hack that never went away.